Documentation/5.4.1/README.rst

```diff
@@ -762,14 +768,14 @@ Create a flavor and profile for computesriov:
     ComputeSriovCount: 2
 
 
-:Step 4: As part of overcloud deployment, Mellanox firstboot template ``/home/stack/templates-original/mellanox_fw_update.yaml`` will be updating firmware on CX5 interface. Create FW folder that will contain all the Mellanox Firmware bin files on a machine that has httpd server running. (User can use the undercloud itself)
+:Step 4: As part of overcloud deployment, Mellanox firstboot template ``/home/stack/nuage-tripleo-heat-templates/firstboot/mellanox_fw_update.yaml`` will be updating firmware on CX5 interface. Create FW folder that will contain all the Mellanox Firmware bin files on a machine that has httpd server running. (User can use the undercloud itself)
 
 ::
 
     $ mkdir -p /var/www/html/FW_<VERSION>
 
```

Comment:
Do we need to ensure that only one firmware version exists in this directory.

Reply:
@dttocs  Can you please elaborate on this more ?